addons/web/static/src/env.js

```diff
@@ -168,3 +169,39 @@ async function _startServices(env, toStart) {
         return null;
     }
 }
+
+/**
+ * Create an application with a given component as root and mount it. If no env
+ * is provided, the application will be treated as a "root": an env will be
+ * created and the services will be started, it will also be set as the root
+ * in `__WOWL_DEBUG__`
+ *
+ * @param {import("@odoo/owl").Component} component the component to mount
+ * @param {HTMLElement} target the HTML element in which to mount the app
+ * @param {Object} [param2] options
+ * @param {Object} [param2.env] an optional environment. If none is provided,
+ *  a new one will be created automatically and the services will be started.
+ * @param {string} [param2.name] a name for the application, will show up in the
+ *  Owl devtools
+ */
+export async function mountComponent(component, target, { env, name } = {}) {
```

Comment:
Defaulting to the component name seems like a decent idea. Since it's only used to display in the owl devtools I'm not in favor of making it mandatory.